Xamarin.Forms.Platform.UAP/DispatcherProvider.cs

```diff
@@ -27,6 +27,11 @@ public IDispatcher GetDispatcher(object context)
 						s_current = new Dispatcher(renderer.ContainerElement.Dispatcher);
 						return s_current;
 					}
+					else
+					{
+						s_current = new Dispatcher();
+						return s_current;
+					}
```

Comment:
This is to fix the scenario where you try and get the dispatcher before the renderer is created, so we fall back to the window's dispatcher. This is because the dispatcher is cached and if there is no renderer, the FallbackDispatcher is used - and that is always using the first/main thread via the Device.